### ISSUE TITLE
autoRun property in task in tasks.json

### ISSUE THREAD

**Issue by lukaszpolowczyk** [philfontaine/autolaunch#1]
Instead this in settings.json:
```
"autolaunch.config": [
		{
			"type": "task",
			"name": "npm start"
		}
	]
```

add support to `"autoRun": true` property in tasks.json:
```
{
			"label": "npmstart",
			"type": "shell",
			"command": "npm start",
			"autoRun": true
		}
```
The effect will be the same and easier to obtain.
What you think?

**Comment by philfontaine** [comment]
Great idea, I will look into it.

**Comment by lukaszpolowczyk** [comment]
For example [vscode-status-bar-tasks](https://github.com/guardrex/vscode-status-bar-tasks) added property "showInStatusBar" for tasks.
You can see him how he did it.

**Comment by philfontaine** [comment]
This feature is now available in the new version 1.0.0. The property is `auto` . Again, thank you for the idea!

**Comment by lukaszpolowczyk** [comment]
Wow great. It went well.
Thanks. :)
